### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by longshorej**
See commit messages.

### CODE REVIEW

File: src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala [hunk 1]
```diff
@@ -63,6 +63,7 @@ sealed trait App extends SbtReactiveAppKeys {
     // I will need to investigate why `akkaClusterBootstrapEnabled` is a task, not a setting.
     enableServiceDiscovery := false,
     akkaClusterBootstrapEndpointName := "akka-remote",
+    akkaClusterBootstrapManagementEndpointName := "akka-mgmt-http",
```
Comment (longshorej) [inline src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala]: There's a 15 character limit on K8s, hence this shortened string

File: src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala [hunk 2]
```diff
@@ -85,11 +86,12 @@ sealed trait App extends SbtReactiveAppKeys {
 
     endpoints := {
       val endpointName = akkaClusterBootstrapEndpointName.value
+      val managementEndpointName = akkaClusterBootstrapManagementEndpointName.value
       val bootstrapEnabled = enableAkkaClusterBootstrap.value.getOrElse(akkaClusterBootstrapEnabled.value)
 
       endpoints.?.value.getOrElse(Seq.empty) ++ {
         if (bootstrapEnabled)
-          Seq(TcpEndpoint(endpointName, 0))
+          Seq(TcpEndpoint(endpointName, 0), TcpEndpoint(managementEndpointName, 0))
```
Comment (longshorej) [inline src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala]: Probably we should add `def apply(name: String)` that defaults the port to `0` to reactive-lib, some other time I think

File: src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala [hunk 3]
```diff
@@ -120,8 +120,13 @@ object SbtReactiveAppPlugin extends AutoPlugin {
         Seq(file)
       }.taskValue,
 
-      javaOptions in SbtNativePackager.Universal ++= Vector(
-        "-Dconfig.resource=app.conf"),
+      javaOptions in SbtNativePackager.Universal ++=
+        Vector(
+          "-Dconfig.resource=app.conf") ++
+          (
+            if (memory.value.isDefined)
```
Comment (fsat) [inline src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala]: Perhaps consider introducing another flag for the user to opt out and disable these options despite having `memory` set?

File: src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala (hunk 1 above)
Comment (fsat) [reply src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala]: Added this to the validation card: https://github.com/lightbend/sbt-reactive-app/issues/13

File: src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala (hunk 3 above)
Comment (longshorej) [reply src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala]: Thanks - raised as https://github.com/lightbend/sbt-reactive-app/issues/31

**Pull request merged**